### PR TITLE
Fix UCX configurations

### PR DESCRIPTION
**Pull request by pentschev**

### CODE REVIEW

File: dask_cuda/tests/test_ucx_options.py [hunk 1]
```diff
@@ -19,14 +19,15 @@
 
 def _test_global_option(seg_size):
     """Test setting UCX options through dask's global config"""
-    dask.config.set(
+    dask.config.update(
```
Comment (quasiben) [inline dask_cuda/tests/test_ucx_options.py]:
```suggestion
    dask.config.update(
        {
            dask.config.global_config,
            "ucx": {
                "SEG_SIZE": seg_size,
                "TLS": "tcp,sockcm,cuda_copy",
                "SOCKADDR_TLS_PRIORITY": "sockcm",
            },
            priority="new"
        }
    )
```

**Comment by pentschev** [comment]
Added a test now, but it will only run if on a DGX machine, therefore codecov will still complain and we will need to ignore that for the time being. Besides that, we're good for a review @quasiben .

**Comment by quasiben** [comment]
Currently failing with:
```
=================================== FAILURES ===================================

___________________________ test_initialize_ucx_tcp ____________________________



    def test_initialize_ucx_tcp():

        p = mp.Process(target=_test_initialize_ucx_tcp)

        p.start()

        p.join()

>       assert not p.exitcode

E       assert not 1

E        +  where 1 = <SpawnProcess(SpawnProcess-15, stopped[1])>.exitcode



dask_cuda/tests/test_initialize.py:51: AssertionError

__________________________ test_initialize_ucx_nvlink __________________________



    def test_initialize_ucx_nvlink():

        p = mp.Process(target=_test_initialize_ucx_nvlink)

        p.start()

        p.join()

>       assert not p.exitcode

E       assert not 1

E        +  where 1 = <SpawnProcess(SpawnProcess-16, stopped[1])>.exitcode

```

**Comment by quasiben** [comment]
This looks good to me. Thank you Peter!

**Comment by pentschev** [comment]
Tests here are failing because it's missing https://github.com/dask/distributed/pull/3539, so we're gonna have to rerun after it gets merged.

**Comment by quasiben** [comment]
rerun tests

**Comment by quasiben** [comment]
rerun test

**Comment by quasiben** [comment]
rerun tests

**Comment by pentschev** [comment]
rerun tests

**Comment by pentschev** [comment]
rerun tests

File: dask_cuda/benchmarks/local_cudf_merge.py [hunk 2]
```diff
@@ -196,13 +196,18 @@ def main(args):
...
+        rmm.reinitialize(
+            pool_allocator=not args.no_rmm_pool,
+            devices=0,
+            initial_pool_size=initial_pool_size,
+        )
         cupy.cuda.set_allocator(rmm.rmm_cupy_allocator)
 
-    client.run(_worker_setup)
+    client.run(_worker_setup, 24e9)
+    client.run_on_scheduler(_worker_setup, 1e9)
```
Comment (quasiben) [inline dask_cuda/benchmarks/local_cudf_merge.py]: I think we should add a comment -- I should've done this before.  Something like:
create pool on scheduler due to occasional deserialization of cuda objects

And the same on local_cupy_transpose_sum.py

**Comment by quasiben** [comment]
This looks great peter!  Thank you.  I left one small one suggestion on adding a comment to the benchmark scripts

File: dask_cuda/benchmarks/local_cudf_merge.py (hunk 2 above)
Comment (pentschev) [reply dask_cuda/benchmarks/local_cudf_merge.py]: Done.

**Comment by quasiben** [comment]
Merging

**Comment by pentschev** [comment]
Thanks @quasiben .

**Pull request merged**